hazelcast/src/main/java/com/hazelcast/executor/impl/DistributedExecutorService.java

```diff
@@ -90,16 +91,21 @@ public void shutdown(boolean terminate) {
     }
 
     public void execute(String name, String uuid, Callable callable, Operation op) {
-        startPending(name);
-        CallableProcessor processor = new CallableProcessor(name, uuid, callable, op);
+        ExecutorConfig cfg = nodeEngine.getConfig().findExecutorConfig(name);
```

Comment:
the problem with `findExecutorConfig` is that it can be very costly. We wouldn't want to call it for each execution. So I suggest a map for caching the found config